src/backend/core/api/viewsets.py

```diff
@@ -332,6 +333,9 @@ def get_queryset(self):
         queryset = super().get_queryset()
         user = self.request.user
 
+        # Annotate the number of accesses associated with each document
+        queryset = queryset.annotate(nb_accesses=Count("accesses"))
```

Comment:
The `nb_accesses` is too big on some documents, adding distinct seems to solve the problem.
```suggestion
        queryset = queryset.annotate(nb_accesses=Count("accesses", distinct=True))
```